forge/build.gradle

```diff
@@ -138,7 +138,7 @@ dependencies {
     modCompileOnly("curse.maven:common-capabilities-247007:4987207")
 
     // Mekanism
-    modCompileOnly ("curse.maven:mekanism-268560:4644795")
+    modImplementation ("maven.modrinth:mekanism:${mekanism_version}")
```

Comment:
Would rather keep this modCompileOnly. Frankly would like to do this to other mods that are included by default but lets not make the bloated dev environment worse than it already is